torch/utils/cpp_extension.py

```diff
@@ -1094,6 +1110,14 @@ def _jit_compile(name,
 
     if verbose:
         print('Loading extension module {}...'.format(name))
+    if translated_files is not None:
+        for fn in translated_files.values():
+            os.unlink(fn)
+            try:
+                os.rmdir(os.path.dirname(fn))
+            except IOError:
+                pass
```

Comment:
Is there a reason you're rm'ing the translated files?

Reply:
So this is a bit of a grey area.
For extensions built from provided source files, we have, in theory, two options where to put the translated files:

- next to the source, which is what we do,
- in the build directors (/tmp/torch_extensions). This runs into trouble when you have includes etc which now are somewhere else.
So I picked the first.

But now we have modified the source directory contents (which is a bit of an annoyance while developing for PyTorch itself, too). We could keep the generated files or not.
The CI seems to enforce that we end with a clean git checkout. So I could either remove them here or remove them in the test.
_Personally_, I'd lean towards keeping the files around. But removing them in the test would make me feel dirty, too.

If you have advice, I'd gladly follow it.

Reply:
I'll tentatively clean up in the test.